### ISSUE TITLE
CMS

### ISSUE THREAD

**Issue by aschwersenz** [ushahidi/opensourcedesign#13]
My suggestion would be to use Wordpress. It is a reasonable CMS for a website like this, the admin area where you manage the content and so on is very user friendly and easy use to understand also for people with few experience with CMS and it enables fast development. I will combine it with a set of plugins, e.g. for improving performance and simplyifing the content editor, that I regularly use. Although Wordpress is free of charge, there might be some miner fees for the plugins (~50-100$). Anyone objecting this choice?

**Comment by aschwersenz** [comment]
@justinscherer Are the plugin costs a 1-time fee? Or are they monthly/annual?

**Comment by aschwersenz** [comment]
1-time

**Comment by aschwersenz** [comment]
There should be budget in our contract with you folks for the plugins,m so that's no issue. cc @Thomas Kueber

**Comment by aschwersenz** [comment]
Do we even need a CMS? Who is going to maintain the WP instance (and plugins/compatibility) once phase 1 is over? Even with auto-updates configured, WP adds significant overhead… The various page elements need to be built, optimised and tested in HTML/CSS/JS anyway, why the added efforts of creating and maintaining a WP theme, and maintaining the whole system?

Just poking holes into that proposal. My gut feeling says we’re probably best off with a set of HTML templates and a static site. While we’re likely to syndicate our content on Medium we would defo want to keep them on our own property as well, but that’s adding an article based off of a template, validating the HTML, and adding a link to the index page, isn’t it?

At this point… I’d rather save some energy for later, in case we might need Andreas to add on a component we didn’t think about at this point in time?

Food for thought. (edited)

**Comment by aschwersenz** [comment]
from Justin: I'm all for keeping it simpler in the short term

**Comment by aschwersenz** [comment]
from Jay: …not even talking about performance :slightly_smiling_face: We use WP for opendevicelab.com (which is unfortunately broken at the moment) but we’re only using it for the backend… added our own API layer that generates static frontend code because slooow.

**Comment by aschwersenz** [comment]
from Eriol: My concern with HTML templates etc. is the time to set up a local host and all the push/pull/versioning of the site will be more 'devvy' so out of a comfort zone here

**Comment by aschwersenz** [comment]
I would recommend using a CMS. The effort on my side whether building static html pages or a wordpress theme is basically the same and it would enable you to easily change content without needing technical assistance for uploading etc. Espacially at this early stage where a lot of things are changing maybe quickly you can easily adapt the website to new things that might arise. And it also in my opinion the more future proof alternative for what might come later. And keeping a wordpress up to date requires only an hour every couple of month maybe. (edited)

**Comment by aschwersenz** [comment]
from Jay: The point is that I really don’t see huge growth in terms of subsections and tons of categories and articles even at later points. But, yes, I see the point of some on the team might feel better with not having to deal with code/repo (wasn’t that the whole point of the OSD project, haha?).

What about the WP performance impact. Is there a reliable and lightweight caching setup you’d have at hand? Basically, I would like to avoid having ANY potential weak spots on the live site, often especially introduced through plugins, and then we don’t have proper administrative monitoring of the site and the shit hits the fan…

Can we use WP in the backend but not expose access to it in the frontend (as we did on opendevicelab.com)?

**Comment by aschwersenz** [comment]
Normaly I have used WP SuperCache so far, some people now prefer WP Rocket, I would take a look at it. I will anyway for the reasons you mentioned keep the usage of plugins to a necessary minimum. “Can we use WP in the backend but not expose access to it in the frontend (as we did on opendevicelab.com)?” I am not a 100% sure what your exact solution there was? Apart from that I will anyway apply safety measures, e.g. disabling the wordpress apis and so on.
